packages/cli/src/commands/run.js

```diff
@@ -0,0 +1,93 @@
+import fs from 'fs'
+import path from 'path'
+
+import babelRequireHook from '@babel/register'
+import Listr from 'listr'
+import VerboseRenderer from 'listr-verbose-renderer'
+import terminalLink from 'terminal-link'
+
+import { getPaths } from 'src/lib'
+import c from 'src/lib/colors'
+
+babelRequireHook({
+  extends: path.join(getPaths().api.base, '.babelrc.js'),
+  extensions: ['.js', '.ts', '.tsx', '.jsx'],
+  only: [getPaths().api.base],
+  plugins: [
+    [
+      'babel-plugin-module-resolver',
+      {
+        alias: {
+          src: getPaths().api.src,
+        },
+      },
+    ],
+  ],
+  ignore: ['node_modules'],
+  cache: false,
+})
+
+const { db } = require(path.join(getPaths().api.lib, 'db'))
+
+const runScript = async (scriptPath) => {
+  const script = await import(scriptPath)
+  const startedAt = new Date()
+  await script.default({ db })
+  const finishedAt = new Date()
+
+  return { startedAt, finishedAt }
+}
+
+export const command = 'run'
```

Comment:
Your positionals will work if you do this
```suggestion
export const command = 'run <name>'
```

Reply:
Ah perfect - this fixed it!